packages/host-arch-ppc64/host-arch-ppc64.1/opam

```diff
@@ -0,0 +1,17 @@
+opam-version: "2.0"
+synopsis: "OCaml on 64-bit IBM POWER"
+description: """
+This package is installed if the underlying OCaml compiler is for
+64-bit IBM POWER.
+
+Precisely, this means `ocamlopt -config-var architecture` equals `power`.
```

Comment:
Just a "nit" question here: why `ppc64` instead of the OCaml `power` term in the package?

Reply:
Indeed - the namespacing here is just horrible from a usability perspective (recalling near errors over riscv vs riscv64, etc.!) - so many different names, and often in strings 😱

I opted that opam-based consistency was the best thing (or "least-worst"), so went with the values opam would use for `%{arch}%` on the basis that both are likely to be used in opam files and while it's odd that `ocaml -config-var architecture` gives `power`, it's probably stranger to have to write `"host-arch-power" {arch = "ppc64"}` in the opam files.

opam's `arch` variable is whatever `uname -m` gives with normalisations as given in `OpamSysPoll.normalise_arch` which pretty much reduce them to x86_64/x86_32, ppc64/ppc32, arm64/arm32, falling back to the `uname -m` value.